commands/bootstrap/resources.go

```diff
@@ -176,6 +184,11 @@ func getVersionFromTOSCATypes(path string) string {
 			version = match[1]
 			return version
 		}
+		err = src.Close()
```

Comment:
When matching close will never be called.

Reply:
I changed this !